README.md

```diff
@@ -64,6 +64,7 @@ URL=http://localhost/3000
 BANNER_TABLE_NAME=local-banners
 REPORT_DATA_TABLE_NAME=local-mcpar-reports-data
 REPORT_TABLE_NAME=local-mcpar-reports
+FORM_TABLE_NAME=local-forms
```

Comment:
i'm guessing we'll want to keep `mcpar` separate from `mlr` and `naaar`, so maybe the name should be `local-mcpar-forms`?

Reply:
updated